### PR TITLE
Update Fulltext index section to not mention B-tree indexes

### PR DESCRIPTION
**Pull request by Hunterness**
Was missed when b-tree indexes was removed

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala [hunk 1]
```diff
@@ -48,7 +48,7 @@ class FulltextIndexTest extends DocumentingTest {
     synopsis("This chapter describes how to use full-text indexes, to enable full-text search.")
     p("""Full-text indexes are powered by the link:https://lucene.apache.org/[Apache Lucene] indexing and search library, and can be used to index nodes and relationships by string properties.
         #A full-text index allows you to write queries that match within the _contents_ of indexed string properties.
-        #For instance, the b-tree indexes described in previous sections can only do exact matching or prefix matches on strings.
+        #For instance, the range and text indexes described in previous sections can only do some matching on strings - exact, prefix, substring or suffix matches.
```
Comment (mnd999) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala]:
```suggestion
        #For instance, the range and text indexes described in previous sections can only perform limited matching on strings - exact, prefix, substring or suffix matches.
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala [hunk 2]
```diff
@@ -71,10 +71,10 @@ class FulltextIndexTest extends DocumentingTest {
         #Using this feature, it is possible to work around the slow Lucene writes from the performance critical commit process, thus removing the main bottlenecks for Neo4j write performance.""".stripMargin('#'))
     p("""At first sight, the construction of full-text indexes can seem similar to regular indexes.
         #However there are some things that are interesting to note:
-        #In contrast to <<administration-indexes-search-performance, b-tree indexes>>, a full-text index can be:""".stripMargin('#'))
+        #In contrast to the <<administration-indexes-search-performance, other indexes>>, a full-text index can be:""".stripMargin('#'))
```
Comment (mnd999) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala]: I'm not sure about 'other' here. Perhaps 'standard' or 'regular'. I'm not sure what's best.

**Review by mnd999 — APPROVED** [review]
Seems fine, I built it and the links are okay. A couple of suggestions.

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala (hunk 2 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala]: Not sure, lets see what the docs team says?

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala [hunk 3]
```diff
@@ -48,7 +48,7 @@ class FulltextIndexTest extends DocumentingTest {
     synopsis("This chapter describes how to use full-text indexes, to enable full-text search.")
     p("""Full-text indexes are powered by the link:https://lucene.apache.org/[Apache Lucene] indexing and search library, and can be used to index nodes and relationships by string properties.
         #A full-text index allows you to write queries that match within the _contents_ of indexed string properties.
-        #For instance, the b-tree indexes described in previous sections can only do exact matching or prefix matches on strings.
+        #For instance, the range and text indexes described in previous sections can only perform limited matching on strings - exact, prefix, substring or suffix matches.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala]:
```suggestion
        #For instance, the range and text indexes described in previous sections can only perform limited matching on strings - exact, prefix, substring, or suffix matches.
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala (hunk 2 above)
Comment (AlexicaWright) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala]: I'd say `regular` or simply drop the article. Prefer the latter.

Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala]:
```suggestion
        #In contrast to <<administration-indexes-search-performance, other indexes>>, a full-text index can be:""".stripMargin('#'))
```

**Review by AlexicaWright — COMMENTED** [review]
Some suggestions. Thank you @Hunterness !

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala (hunk 2 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala:74]: so something like this?
```suggestion
        #In contrast to <<administration-indexes-search-performance, other indexes>>, a full-text index can be:""".stripMargin('#'))
```

Comment (AlexicaWright) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala]: Exactly, I did leave an in-line suggestion too, looks exactly like this :) .

**Review by AlexicaWright — APPROVED** [review]
LGTM 👍  Thank you @Hunterness !

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala (hunk 2 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala]: the fact that I removed my comment when I noticed your suggestion makes this comment thread interesting now :P

**Pull request merged**